airbyte-commons/src/main/resources/log4j2.xml

```diff
@@ -1,28 +1,60 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<Configuration status="INFO">
+<Configuration status="TRACE">
+<!--<Configuration status="INFO">-->
     <Properties>
         <Property name="default-pattern">%d{yyyy-MM-dd HH:mm:ss}{GMT+0} %highlight{%p} %C{1.}(%M):%L - %X - %m%n</Property>
         <Property name="default-worker-file-pattern">%d{yyyy-MM-dd HH:mm:ss}{GMT+0} %p (%X{job_root}) %C{1}(%M):%L - %m%n</Property>
+
+        <Property name="s3-bucket">$${env:S3_LOG_BUCKET}</Property>
+        <Property name="s3-region">$${env:S3_LOG_BUCKET_REGION}</Property>
+        <Property name="s3-aws-key">$${env:AWS_ACCESS_KEY_ID}</Property>
+        <Property name="s3-aws-secret">$${env:AWS_SECRET_ACCESS_KEY}</Property>
+
+        <Property name="gcp-storage-bucket">$${env:GCP_STORAGE_BUCKET}</Property>
+
     </Properties>
 
     <Appenders>
         <Console name="Default" target="SYSTEM_OUT">
             <PatternLayout pattern="${default-pattern}"/>
         </Console>
+
         <Routing name="LogSplit">
             <Routes pattern="$${ctx:job_root}">
                 <!-- Don't split logs if job_root isn't defined -->
                 <Route key="$${ctx:job_root}">
                     <Null name="/dev/null"/>
                 </Route>
                 <Route>
-                    <File name="${ctx:job_root}" fileName="${ctx:job_root}/${ctx:job_log_filename}">
+                    <File name="${ctx:job_root}-local" fileName="${ctx:job_root}/${ctx:job_log_filename}">
                         <PatternLayout pattern="${default-worker-file-pattern}"/>
                     </File>
                 </Route>
             </Routes>
             <IdlePurgePolicy timeToLive="15" timeUnit="minutes"/>
         </Routing>
+        <Routing name="LogSplitCloud">
```

Comment:
will comment why we had to configure in this way - essentially <Route> only accepts one appender.